cmake/Platform/Linux/LYWrappers_linux.cmake

```diff
@@ -52,19 +62,48 @@ endfunction()
 #! ly_apply_debug_strip_options: Apply debug stripping options to the target output for non-debug configurations.
 #
 #\arg:target Name of the target to perform a post-build stripping of any debug symbol)
-function(ly_apply_debug_strip_options target)
+#\arg:target_type The target's output type. STATIC, SHARED, EXECUTABLE, and STATIC are supported types.
+function(ly_apply_debug_strip_options target target_type)
```

Comment:
We can query the target type inside of this function instead of needing to pass it in.